### ISSUE TITLE
Project fails to launch/compile when VRM4U is added.

### ISSUE THREAD

**Issue by talnacialex** [ruyo/VRM4U#33]
Hello. I wanted to try out VRM4U in an Unreal Project.

I'm using Unreal Engine v4.24.3 and the latest VRM4U Unreal Extension(the one from the private repo).
I managed to add it, but when I compile with it, things go wrong. I tried it even in an fresh project, but it gives me this error:

> LogPlayLevel: Error:   ERROR: Could not find definition for module 'VRM4U', (referenced via CodenameSakura.uproject -> VRM4U.uplugin)

The plugin is added in the project plugins folder. I have Visual Studio 2017 Community installed and configured(there is also VS2019 Professional).

This is my log:
[CodenameSakura.log](https://github.com/ruyo/VRM4U/files/4884631/CodenameSakura.log)

Could you please help me or tell me what I'm doing wrong?
Thank you.

**Comment by ruyo** [comment]
Hi. Thanks for trying.

Just in case, can you check if the following files exist?
CodenameSakura/Plugins/VRM4U/Source/VRM4U/VRM4U.Build.cs

Also try running GenerateProjectFiles again.
I feel like there are not enough files in that environment.

**Comment by talnacialex** [comment]
I think I got it fixed. I tried some more new projects.
This may be useful for others that hurry up(that was also the case of mine)

What I did differently was to not install it IN the engine.
Instead, this is what I did step by step:

1. Got VRM4U from this repo for my Unreal version
2. Unzipped the Plugins folder into my project directory
3. Started Unreal, added my characters
4. Did the retargeting and everything else.
5. Closed Unreal. Got the source version of VRM4U(from the private repo) and I drag&dropped all directories in it(so in my project root there are 2 folders: Plugins and releasetool(idk if needed)).
6. Started once again Unreal
7. Created an C++ class(from File -> New C++ Class) - selected None(and left everything default)
8. Then everything went fine.

Thanks for the support and good luck!
